lib/core/src/Router.ts

```diff
@@ -0,0 +1,42 @@
+import { Param } from '@exteranto/ioc'
+import { Script } from '@exteranto/support'
+
+export class Router {
+  /**
+   * The current script.
+   *
+   * @var {string}
+   */
+  @Param('script')
+  private static script: Script
+
+  /**
+   * The routes to be added.
+   *
+   * @var {any[]}
+   */
+  private static routes: any = []
+
+  /**
+   * Add routes to the global collection.
+   *
+   * @param {any[]} routes
+   * @param {Script} script
+   */
+  public static add (routes: any[], script?: Script) : typeof Router {
+    if (script === undefined || script === this.script) {
+      this.routes = this.routes.concat(routes)
```

Comment:
If we're going to do the `edit` functionality, I reckon
`this.routes.push(...routes)`
is preferable?